### PR TITLE
tests: add more cases for core-flags tests

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
tests

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
No
**Summary**
increase coverage.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by evilebottnawi — DISMISSED** [review]

File: test/core-flags/entry-reset-flag.test.js [hunk 1]
```diff
@@ -11,4 +11,12 @@ describe('--entry-reset flag', () => {
         expect(stdout).toContain('src/entry.js');
         expect(stdout).not.toContain('src/main.js');
     });
+
+    it('should thorw error if entry is an empty array', () => {
```
Comment (jamesgeorge007) [inline test/core-flags/entry-reset-flag.test.js:15]:
```suggestion
    it('should throw error if entry is an empty array', () => {
```

**Review by rishabh3112 — DISMISSED** [review]

**Review by jamesgeorge007 — DISMISSED** [review]
A minor typo left out, and we are good to go.

File: test/target/flag-test/target-flag.test.js [hunk 2]
```diff
@@ -63,5 +63,21 @@ describe('--target flag', () => {
...
+
+        it('should reset target from node to async-node with --target-reset', () => {
+            const { stderr, stdout, exitCode } = run(__dirname, ['--target-reset', '--target', 'async-node']);
+
+            expect(exitCode).toBe(0);
+            expect(stderr).toBeFalsy();
+            expect(stdout).toContain(`target: [ 'async-node' ]`);
+        });
+
+        it('should thorw error if target is an empty array', () => {
```
[the remark this answers is not in the document]
Comment (snitin315) [reply test/target/flag-test/target-flag.test.js:75]: thanks!

**Review by jamesgeorge007 — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Pull request merged**